src/shared/components/QuickLaneLink/QuickLaneLink.tsx

```diff
@@ -22,7 +22,12 @@ const QuickLaneLink: FunctionComponent<QuickLaneLinkProps> = ({ id, label, short
 	});
 
 	return (
-		<a className={className} target="_blank" href={generateQuickLaneHref(id)}>
+		<a
+			className={className}
+			target="_blank"
+			rel="noopener noreferrer"
```

Comment:
In theorie hoef je `noopener` niet meer te definiëren als `noreferrer` gedefiniëerd is. [docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer)